rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -569,70 +701,83 @@ extern "C" RocalTensor ROCAL_API_CALL rocalCropMirrorNormalize(RocalContext cont
                                                                RocalTensorOutputType rocal_tensor_output_datatype = ROCAL_UINT8);
 
 extern "C" RocalTensor  ROCAL_API_CALL rocalCrop(RocalContext context, RocalTensor input, bool is_output,
-                                             RocalFloatParam crop_width = NULL,
-                                             RocalFloatParam crop_height = NULL,
-                                             RocalFloatParam crop_depth = NULL,
-                                             RocalFloatParam crop_pox_x = NULL,
-                                             RocalFloatParam crop_pos_y = NULL,
-                                             RocalFloatParam crop_pos_z = NULL);
+                                                 RocalFloatParam crop_width = NULL,
+                                                 RocalFloatParam crop_height = NULL,
+                                                 RocalFloatParam crop_depth = NULL,
```

Comment:
is crop_depth, crop_pos_z relevant in existing api too?

Reply:
Not using, might need in future